glean-core/android/src/test/java/mozilla/telemetry/glean/DispatchersTest.kt

```diff
@@ -60,4 +65,52 @@ class DispatchersTest {
         assertEquals("Tasks have executed", 3, threadCanary)
         assertEquals("Task queue is cleared", 0, Dispatchers.API.taskQueue.size)
     }
+
+    @Test
+    fun `queued tasks are executed in the order they are received`() {
+        val orderedList = mutableListOf<String>()
```

Comment:
Well, I guess I would have gone for a list of ints, but converting them to a string is just as fine

Reply:
Probably lighter weight yet to go with Ints, point taken.